src/serialize.jl

```diff
@@ -0,0 +1,18 @@
+const pickle = PyNULL()
+
+function get_pickle()
+    if pickle.o == C_NULL
+        copy!(pickle, pyimport("pickle"))
+    end
+
+    return pickle
+end
+
+function Base.serialize(s::AbstractSerializer, pyo::PyObject)
+    Base.serialize_type(s, PyObject)
+    serialize(s, get_pickle()[:dumps](pyo))
```

Comment:
In particular:
```
b = PyBuffer(pycall(p["dumps"], PyObject, [1,2,3]))
serialize(s, unsafe_wrap(Array, Ptr{UInt8}(b.buf.buf), b.buf.len))
```
should be about as good as we can get without dropping down deeper into the `serialize` internals to work with raw pointers.